scarb/src/compiler/plugin/proc_macro/ffi.rs

```diff
@@ -73,6 +73,20 @@ impl ProcMacroInstance {
         })
     }
 
+    /// Loads a procedural macro from a prebuilt shared library.
+    pub fn try_load_prebuilt(package: Package, _config: &Config) -> Result<Self> {
```

Comment:
This only differs by path from `ProcMacroInstance::try_new`, right? Can we unify those two then (i.e. just accept path as argument)?